### DISCUSSION TITLE
Render a Quarto book without evaluating code chunks.

### DISCUSSION THREAD

**Discussion by karaesmen** [quarto-dev/quarto-cli#7759] [Q&A] [answered]
### Description

I am writing a quarto book, and some code chunks take a while to render. I want to render the entire book to see and resolve formatting issues without evaluating any of the code chunks. I tried adding the following to my `_quarto.yml` file bt didn't work.

```
execute:
  eval: false
```

 Is there another way to achieve that?

**Comment by karaesmen** [comment, marked as answer]
The code chunks that had `eval: true` was overriding the yml parameter and causing the problem. After removing them all with `Find in Files`, it works!

**Reply by cscheid** [reply]
Just to clarify: this behavior is by design. In general, the principle is that, more specific context settings override more general contexts, from project-level to document-level to element-level.
